core/src/main/scala/chisel3/RawModule.scala

```diff
@@ -154,6 +180,44 @@ package object internal {
   /** Marker trait for modules that are not true modules */
   private[chisel3] trait PseudoModule extends BaseModule
 
+  /** Computes a name
+    *
+    * @param prefix The prefix associated with the seed
+    * @param seed The seed for computing the name (if available)
+    * @param defaultPrefix Optional default prefix which is only used if the seed is not defined
+    * @param defaultSeed Optional default (or "fallback") seed
+    * @return the name, if it can be computed
```

Comment:
is it `None` if it cannot be computed? Under what scenarios can it not be computed?

Reply:
It can't be computed if seed and default seed are both `None`.